src/include/daos/object.h

```diff
@@ -135,6 +135,8 @@ struct daos_obj_md {
 	 * use pl_map's default value PL_DEFAULT_DOMAIN (PO_COMP_TP_RANK).
 	 */
 	uint32_t		omd_fdom_lvl;
+	/* Performance domain affinity */
+	uint32_t		omd_pda;
```

Comment:
not issue of this patch, but the comment "object metadata stored in the global OI table of container" is out-of-date, it's not in OI table

Reply:
right, I was confused when read that comment too. can remove it later in following patch.